app/_locales/en/messages.json

```diff
@@ -556,6 +556,12 @@
   "dismiss": {
     "message": "Dismiss"
   },
+  "dismissReminderDescriptionField": {
+    "message": "Turn this on to dismiss the seed-phrase backup reminder message. We highly recommend that you back up your seed phrase to avoid loss of funds"
```

Comment:
This is placeholder text. @rachelcope do you want to rethink this verbiage?

Reply:
This looks good. We are updating "seed phrase" to "recovery phrase" so we might want to update that here as well